src/RuleSet/Loader/Composer/Package.php

```diff
@@ -42,7 +42,7 @@ public static function fromArray(array $package)
      */
     public function generatePackageKey()
     {
-        return str_replace('/', '_', $this->name).'_'.$this->version;
+        return str_replace(['/', '\\', ':'], '_', $this->name).'_'.$this->version;
```

Comment:
The short array syntax is not compatible with PHP 5.3 (see the failing Travis build).